### ISSUE TITLE
Tuplets shown wrong

### ISSUE THREAD

**Issue by dror12345** [opensheetmusicdisplay/opensheetmusicdisplay#485]
Hi all, 
Maybe I got something wrong here, but when I use finale to create this kind of notes:
![image](https://user-images.githubusercontent.com/9460568/52568275-f4a31f00-2e16-11e9-8b45-260da3a3eae0.png)

Export them to XML and use your demo to view it, I get this.
![image](https://user-images.githubusercontent.com/9460568/52568301-0ab0df80-2e17-11e9-8823-1d471e543033.png)

I seems that a quarter and then an eight note within a tuplets, causes it to show the number 2 instead of 3 above the notes.

Any idea why is it so?

**Comment by sschmidTU** [comment]
That is indeed a bug.
I corrected it already, on the develop branch. We took the wrong value for the tuplet label.
Fix will be in the next release, thanks for reporting!

Here are a few tuplet tests, relevant for this issue is the one with a triplet consisting of a 4th and 8th.
[tuplet_tests_4th8thtriplet_dotted.zip](https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/files/2861525/tuplet_tests_4th8thtriplet_dotted.zip)

There is still an issue that dotted notes in tuplets do not show a dot, but that will be another issue, as it's also a rarer case.

By the way, try `osmd.setOptions({tupletsRatioed: false});` and reload the sample (or re-render).
